docs/extensibility/available-filters.md

```diff
@@ -127,26 +127,24 @@ There is a snackbar at the bottom of the page used to display notices to the cus
 
 <img src="https://user-images.githubusercontent.com/5656702/120882329-d573c100-c5ce-11eb-901b-d7f206f74a66.png" width=300 />
 
-It may be desirable to hide this (by removing it from the array) or to change the text in the notice.
+It may be desirable to hide this if there's a notice you don't want the shopper to see.
 
 | Filter name  | Description | Return type  |
 |---|---|---|
-| `snackbarNotices`  | An array of notices waiting to be shown in the snackbar | `SnackbarNotice[]`
+| `snackbarNoticeVisibility`  | An object keyed by the content of the notices slated to be displayed. The value of each member of this object will initially be true. | `object`
 
-These are the relevant members of a `SnackbarNotice` object.
+The filter passes an object whose keys are the `content` of each notice.
 
-```typescript
-SnackbarNotice {
-  content: string;
-  explicitDismiss: boolean;
-  icon: string | null;
-  isDismissable: boolean;
-  onDismiss: Function;
-  spokenMessage: string;
-  status: string;
-  type: string
+If there are two notices slated to be displayed ('Coupon code "10off" has been applied to your basket.', and 'Coupon
+code "50off" has been removed from your basket.'), the value passed to the filter would look like so:
```

Comment:
The break between lines 138 and 139 looks to haven been added accidentally. (That said, when looking closer, I see that other sections also show line breaks at places where they should not belong.)

Reply:
I added the linebreaks there because my IDE indicates a "maximum line length" of 120
Here's what I see
![image](https://user-images.githubusercontent.com/5656702/127118371-02509ac8-5712-45be-9fbf-9c311236aa43.png)

The thin line on the right is the 120 character mark. (Tables have to exceed it or their formatting breaks)

Reply:
> I added the linebreaks there because my IDE indicates a "maximum line length" of 120

Ah, I see what you mean. For source code, I'd stick to the "maximum line length", while for *.md files, I'd ignore it and keep paragraphs in one line, in case one if viewing the *.md file in a different reader, e.g. in an IDE. That said, when viewing the *.md file in GitHub, no strange line break is visible. I'll mark this feedback as resolved. 😀